src/mappings/fspiopiso20022/quotes.ts

```diff
@@ -121,6 +125,27 @@ export const quotes_reverse = {
     "body.CdtTrfTxInf.ChrgsInf.Amt.ActiveOrHistoricCurrencyAndAmount": "body.payeeFspFee.amount",
     "body.CdtTrfTxInf.VrfctnOfTerms.IlpV4PrepPacket": "body.ilpPacket"
   }`,
+  put: `{
+    "$noDefaults": "true",
+    "body.GrpHdr.MsgId": { "$transform": "generateID" },
+    "body.GrpHdr.CreDtTm": { "$transform": "datetimeNow" },
+    "body.GrpHdr.NbOfTxs": { "$transform": "fixed", "value": "1" },
+    "body.GrpHdr.SttlmInf.SttlmMtd": { "$transform": "fixed", "value": "CLRG" },
+    "body.GrpHdr.PmtInstrXpryDtTm": "body.expiration",
+    "body.CdtTrfTxInf.PmtId.TxId": "params.ID",
+    "body.CdtTrfTxInf.Dbtr": "$context.isoPostQuote.CdtTrfTxInf.Dbtr",
+    "body.CdtTrfTxInf.DbtrAgt": "$context.isoPostQuote.CdtTrfTxInf.DbtrAgt",
+    "body.CdtTrfTxInf.Cdtr": "$context.isoPostQuote.CdtTrfTxInf.Cdtr",
+    "body.CdtTrfTxInf.CdtrAgt": "$context.isoPostQuote.CdtTrfTxInf.CdtrAgt",
+    "body.CdtTrfTxInf.ChrgBr": "$context.isoPostQuote.CdtTrfTxInf.ChrgBr",
```

Comment:
I believe the initial reason why `$context.isoPostQuote` was needed is to cater for use cases in the sdk-scheme-adapter where some values in the previous post quote are needed for the put quote request. In other services, this is not needed, rather values in the `headers` are used instead. Using the `$alt` transform covers all those use cases in a single mapping. Can we revisit that solution instead?